Sources/StreamChat/Models/CurrentUser.swift

```diff
@@ -52,12 +52,16 @@ public class CurrentChatUser: ChatUser {
     
     /// The unread counts for the current user.
     public let unreadCount: UnreadCount
+
+    /// A Boolean value indicating if the user has opted to hide their online status.
+    public let isInvisible: Bool
```

Comment:
Are we sure this value will always exist? Would be nice to check with BE

Reply:
Yes, either way it has a default value of false